docs/source/dev_arch.rst

```diff
@@ -68,18 +68,75 @@ This choice of architecture means that logging to disk is a little bit more comp
 need to avoid writing to the same log file from multiple processes, to avoid overwriting
 each other‘s logs. We have therefore implemented a log thread and queues delivering log 
 messages from the processes. This means that in a log file, the order of log messages 
-can be out of order.  
+can be out of order. 
+
+To set this up, in each process you need to invoke logging like this:
+
+.. code-block::
+
+    from PiFinder.multiproclogging import MultiprocLogging
+    
+    # You can create loggers with-out setting up forwarding
+    logger = logging.getLogger(„Solver“)
+    
+    ...
+    
+    # In the main loop of the process ... 
+    def process( ..., log_queue, ...)
+        MultiprocLogging.configurer(log_queue) # ... Enable log forwarding
+        
+        # only then create log messages
+        logger.debug(„Starting Solver“)
+
 
 Choice of Plate-Solver
 ------------------------ 
 
-PiFinder uses 
+PiFinder uses ``cedar-detect-server <https://github.com/smroid/cedar-detect>``_ 
+in binary form to determine star centroids in an image. This is a fast centroider written
+in the Rust programming language that is running in a separate process. A gRPC API is used
+to interface with this process. 
 
+The detected centroids are then passed to the 
+``tetra3 solver <https://github.com/esa/tetra3>``_ for plate-solving. 
```

Comment:
This is not quite accurate. The plate solving is done by Cedar-solve, which is a fork of Tetra3.

Reply:
We should definitely update this text as that will be the case by the time this is released. This switch to Cedar Solve will be done with PR #256 which I'll be merging in over the break and will be part of the same release this PR will be a part of.  :+1:

Reply:
Ah, I didn't realize PiFinder is still using o.g. Tetra3.

Reply:
Yeah... I've been... slow on the switch 😅  Mainly because I need to make sure the whole sub-repo switch, new binary for detect, etc all work during the software update without borking everything.  The PR mentioned seems to do this properly so the next software update will be where we *finally* make the full switch 🎉

Reply:
Oh, and to clarify, it's using a commit you PR'd to the tetra3 repo, so it is using effectively an early version of Cedar Solve 🙌  Just before it was a fork

Reply:
Ok, I'll check the respective PR.